plugins/hls-floskell-plugin/src/Ide/Plugin/Floskell.hs

```diff
@@ -33,7 +33,7 @@ provider _ideState typ contents fp _ = liftIO $ do
     config <- findConfigOrDefault file
     let (range, selectedContents) = case typ of
           FormatText    -> (fullRange contents, contents)
-          FormatRange r -> (r, extractRange r contents)
+          FormatRange r -> (normalize r, extractRange r contents)
```

Comment:
Why this change? Is that a bug, where the fix doesn't actually belong here?

Reply:
The test case "formats a range" failed.
I didn't find out the reason, but all other formatting plugins use `normalize`, and this fixed the problem.

Reply:
good enough for me